include/rdma/fi_domain.h

```diff
@@ -296,6 +298,11 @@ struct fi_ops_domain {
 
 /* Memory registration flags */
 /* #define FI_RMA_EVENT		(1ULL << 56) */
+#define FI_HMEM_HOST_ALLOC		(1ULL << 45)
+#define FI_HMEM_DEVICE_ONLY		(1ULL << 46)
+
+/* libfabric internal flags starting from 60 to 63 */
+#define FI_HMEM_DATA_GDRCOPY_HANDLE	(1ULL << 60)
```

Comment:
Hi @shefty I set this flag to 60 based on your prior comment of internal flags. Please let me know if you prefer a different value/name.

Reply:
You are still changing the public header files.  These changes need to be kept internal

Reply:
Ack. Addressed in updated.